src/components/disclosure/disclosure.module.css

```diff
@@ -19,17 +19,18 @@
 /*
 ***
 * Notes:
-*   - Setting only the default state of the .content container prevents us from
-*     accidentally setting the open state of the container to something
-*     incorrect that can potentially break the semantics of the element
-*   - Using the aria-expanded attribute selector ensures we're correctly setting
-*     the value in the component's JS
-*   - Using the next-sibling combinator (`+`) ensures that the .content
-*     container is the sibling element immediately after the button
+*   - Setting only the default (closed) state of the .content container prevents
+*     us from accidentally setting the open state of the container to something
+*     incorrect that can potentially break the semantics of the element.
+*   - Using the next-sibling combinator (`+`) could help ensure the .content
+*     container is the sibling element immediately after the button. It could
+*     also allow us to use the aria-expanded attribute as a CSS selector.
+*     However, when wrapping the button element in a heading element to meet
+*     the Accordion pattern, the sibling selector no longer works.
 *
 * ref: https://adrianroselli.com/2020/05/disclosure-widgets.html#CSS
 ***
 */
-.activator[aria-expanded='false'] + .contentContainer {
+.closedContentContainer {
```

Comment:
🚧 We should retain applying styles using selectors here. Their purpose is to make sure styles are only applied when the programmatic state & DOM order are correct AKA when accessibility is coded correctly.

Reply:
It makes sense that we'd ideally want to do this!

When wrapping in a `<heading>` element, however, I think it's not feasible for us to use the `[aria-expanded='false']` sibling selector, as the `button` should be the only element within the `heading`. We could potentially set `aria-expanded ='false'` on the heading so that the style still works, but it seems like that might really be what we're looking for 🤔 

For clarity, the use of this selector is an extra guarantee, rather than a key part of the functionality, right? In that the DOM order can be correct without the use of this selector? It seems like we're already validating that the DOM order is correct using `validate-disclosure-children`, so given the difficulty in retaining the sibling selector when wrapping `button` in `<h{n} />` to match [the Accordion pattern](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/), my thought was this might be an acceptable compromise in order to get the functionality we need.

Reply:
✅ update - since dropping the `heading` functionality, have retained this selector pattenr.